README.md

```diff
@@ -1,159 +1,158 @@
-  <img src="https://img.shields.io/opencollective/backers/tuistapp?style=flat-square" alt="Backers">
-  <img src="https://img.shields.io/github/license/tuist/tuist?style=flat-square" alt="License">
-  <img src="https://img.shields.io/badge/Powered%20by-Tuist-blue" alt="Powered by Tuist">
-  <a href="https://polar.sh/tuist"><img src="https://polar.sh/embed/seeks-funding-shield.svg?org=tuist" /></a>
-</div>
-
-<div align="center">
-  <a href="https://cal.com/team/tuist/cloud?utm_source=banner&utm_campaign=oss" target="_blank"><img alt="Book us with Cal.com" src="https://cal.com/book-with-cal-dark.svg" /></a>
+  <a href="https://tuist.io" target="_blank">
+    <picture>
+      <img src="assets/header.jpg" alt="header"/><br>
+    </picture>
+  </a>
 </div>
 
-## What's Tuist 🕺
+## 🕺 What's Tuist
 
-Tuist is a command line tool that helps you **generate**, **maintain** and **interact** with Xcode projects.
+Tuist is a command line tool that helps you **_generate_**, **_maintain_** and **_interact_** with Xcode projects.
 
 It's open source and written in Swift.
 
-## Install ⬇️
+## ⬇️ Install
 
-### Recommended: [mise](https://github.com/jdx/mise)
-
-Install [mise](https://mise.jdx.dev/getting-started.html#quickstart) and then run the following command to install Tuist:
-
-```bash
-mise install tuist
-```
+The recommended installation method is to [install mise](https://mise.jdx.dev/getting-started.html#quickstart) and then run `mise install tuist` to install Tuist.
 
 You can check out [the documentation](https://docs.tuist.io/documentation/tuist/installation) to learn more about the rationale behind our installation approach and alternative approaches.
 
-## Bootstrap your first project 🌀
+## 🌀 Bootstrap your first project
 
 ```bash
 tuist init --platform ios
 tuist generate # Generates Xcode project & workspace
 tuist build # Builds your project
 ```
 
-[Check out](https://docs.tuist.io/documentation/tuist) the project "Get Started" guide to learn more about Tuist and all its features.
+Check out [the project "Get Started" guide](https://docs.tuist.io/documentation/tuist) to learn more about Tuist and all its features.
 
-## Sample projects 🔬
+## 🔬 Sample projects
 
 You can find some sample projects in the [fixtures folder](fixtures) or the [awesome Tuist repo](https://github.com/tuist/awesome-tuist)! 🎉
 
-## Want to contribute?
-
-You can use our [contribution docs](https://docs.tuist.io/documentation/tuist/get-started-as-contributor) to get started. If you don't have a specific issue in mind, we are more than happy to help you, just ask for help in a given issue or on our [Slack](https://join.slack.com/t/tuistapp/shared_invite/zt-1lqw355mp-zElRwLeoZ2EQsgGEkyaFgg). You can find good issues for first-time contributors [here](https://github.com/tuist/tuist/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). We also offer [issue bounties](https://github.com/tuist/tuist/discussions/4982) for some highly-valued issues.
-
```

Comment:
I'm a bit on the fence with this one. READMEs are often the entry point from where a person navigates. We might have different types of users coming across this file:
- I'm a developer, I heard of Tuist for the first time and I want to start using it in my project.
- I'm an organization, I heard of Tuist for the first time, and I want to assess the impact in my project.
- I'm a user of Tuist, and had an idea for something to contribute

I think 1 and 2 are addressed, and we need something for 3.